pilot_apps/camunda/openshift/camunda_deployconfig.yaml

```diff
@@ -173,21 +180,22 @@ objects:
               value: ${APP_ROOT_LOG_FLAG}
             - name: BPM_HISTORY_LEVEL
               value: ${BPM_HISTORY_LEVEL}
-            - name: CAMUNDA_MAX_POOLSIZE
-              value: ${CAMUNDA_MAX_POOLSIZE}
+            - name: HIKARI_CONN_TIMEOUT
+              value: ${HIKARI_CONN_TIMEOUT}
+            - name: HIKARI_MAX_POOLSIZE
+              value: ${HIKARI_MAX_POOLSIZE}
             - name: SESSION_STORE_TIMEOUT
               value: ${SESSION_STORE_TIMEOUT}
-            - name: CAMUNDA_CONN_TIMEOUT
-              value: ${CAMUNDA_CONN_TIMEOUT}
-            image: docker-registry.default.svc:5000/${TOOLS_WORKSPACE}/${NAME}:${IMAGE_TAG}
+            - name: SESSION_STORE_TYPE
+              value: ${SESSION_STORE_TYPE}
             imagePullPolicy: Always
             livenessProbe:
               failureThreshold: 3
               httpGet:
                 path: /camunda/actuator/health
                 port: 8080
                 scheme: HTTP
-              initialDelaySeconds: 120
+              initialDelaySeconds: 60
```

Comment:
Lowered this, since it usually starts in ~35s. We can tweak if server patching slowness indicates that it's a little short.